src/components/ChattingScreen.js

```diff
@@ -18,14 +21,96 @@ export default function ChattingScreen() {
 		{ user: false, content: '나도 아니까  ' },
 		{ user: false, content: '그만 말해줘도 돼' },
 		{ user: true, content: 'ㅠㅠ' },
-	];
+	]);
+	function changeSender(e){
+		e.preventDefault();
+		setCurrentSender(!currentSender);
+	}
+	function changeNewMsg(e){
+		e.preventDefault();
+		setNewMsg({user: currentSender, content: e.target.value});
+	}
```

Comment:
의미상 맞는 이름이고 좋은 것 같아요 ㅎㅎ. 다만 함수를 보았을 때 이게 어떤 이벤트로부터 오는지 이름에 더 잘 드러났으면 좋지 않을까 싶어요! 이벤트를 수신하는 이벤트 핸들러라는 것을 더 강조한 네이밍이 좋아 보여요ㅎㅎ

```suggestion
	function handleSenderButtonClick(event){
		event.preventDefault();
		setCurrentSender(!currentSender);
	}
	function handleNewMessageInputChange(event){
		event.preventDefault();
		setNewMsg({user: currentSender, content: e.target.value});
	}
```

Reply:
맞는 말이세요!! 앞으로도 이벤트 핸들러 이름을 지을 때 꼭 handler라는걸 강조해줘야겠어요!!